src/docs.md

```diff
@@ -69,6 +69,15 @@ If for some reason you don't like the global `cadenza` field, you can remove it
 const cadenza = window.cadenza.noConflict();
 ```
 
+### Communication with parent Cadenza
```

Comment:
The window. is confusing, imo. Also I'd expect this whole info in the constructor JSDoc.

Maybe we should mention that fact that a method works with opener and parent in its JSDoc?

Reply:
`window.` is also in all other examples right above it, maybe it looks weird here in the review out-of-context. It makes sense if you look at it rendered. I will also add the info to the constructor JSDoc.